README.md

```diff
@@ -1,6 +1,11 @@
-# rekor-monitor
+# Rekor Log Monitor
+
+Rekor Log Monitor provides an easy-to-use monitor to verify log consistency,
+that the log is immutability and append-only. Monitoring is critical to
+the transparency log ecosystem, as logs are tamper-evident but not tamper-proof.
+
+To run, simply fork this repository, and you'll begin to monitor Rekor.
```

Comment:
that's it just fork it?

Reply:
is it run as an action? Do they have to bring up infra to do so?

Reply:
It runs on GHA! Like https://github.com/sigstore/rekor-monitor/actions/runs/3379989706

I added some details